### PR TITLE
Gives all nuclear bombs unique codes

### LINKED ISSUE #9430: Nukies need an incentive to use their own nuke and to evac the station before it explodes.
## Description
Nuclear operatives need an incentive to use their own nuke aboard their shuttle and not the one in the vault. They also should be incentivized to not suicide by team deathmatch on the station after the nuke is armed.

This would add additional layers to the gamemode and can potentially make it more interesting.
Would need a way to prevent crewmembers from boarding the shuttle if parked close. Or a way for operatives to find their way back to the shuttle if parked in deep space away from the station.

**Additional context**
The current meta is to have the nukies rush the vault nuke after obtaining the disk, once armed, they camp the vault until detonation instead of retreating back to their shuttle. 

Relates to issue #8932; nukies should have additional greentext based on how well they do. Full greentext should only be given to operatives that successfully evacuate the station after nuking it.

Might only be useful in MRP.

All this is purely my opinion, but i thought it would be beneficial for the future of the game to discuss additional ways the gamemode could be played out.

### PR DESCRIPTION
**Pull request by vulppine**
## About the PR 

Changes it so that nuclear bombs now have unique labels, and unique codes for each bomb. This should make it harder for nuclear operatives to blow up the station, as now they have to bring their nuke, instead of using the station's nuclear bomb.

This should close #9430.

:cl:
- tweak: NanoTrasen has finally upgraded all of our nuclear bombs! They no longer all use the exact same code.

### CODE REVIEW

File: Content.Server/Nuke/NukeSystem.cs [hunk 1]
```diff
@@ -406,6 +426,18 @@ private void PlayNukeKeypadSound(EntityUid uid, int number, NukeComponent? compo
...
+        {
+            var ret = "";
+            for (var i = 0; i < length; i++)
+            {
+                var c = (char) _random.Next('0', '9' + 1);
+                ret += c;
+            }
+
+            return ret;
+        }
```
Comment (moonheart08) [inline Content.Server/Nuke/NukeSystem.cs:439]: 🤨 

wouldn't `Next(0,Pow(10, length))` work just fine.

Comment (vulppine) [reply Content.Server/Nuke/NukeSystem.cs:439]: copy pasted from the old nuke code system code 😎

Comment (vulppine) [reply Content.Server/Nuke/NukeSystem.cs:439]: an actual answer: the old nuke code system code seemed to just generate a string of some length made of numbers using this, and i'm just reusing that same code to generate both the nuke code (which was also stored as a string, which is fine enough) and the serial number for nuclear bombs

**Review by mirrorcult — CHANGES_REQUESTED** [review]

**Review by mirrorcult — APPROVED** [review]

**Comment by 0x6273** [comment]
This fixes #9430
(GitHub pro tip: put "closes #issue" in pr desc and it will close it automatically when it's merged)

**Comment by vulppine** [comment]
> This fixes #9430 (GitHub pro tip: put "closes #issue" in pr desc and it will close it automatically when it's merged)

i just wasn't aware that the issue existed, oops (i probably should've searched for related issues)

**Pull request merged**